auction_keeper/gas.py

```diff
@@ -83,7 +83,8 @@ def __str__(self):
         else:
             retval = f"Default gas 10 Gwei "
 
-        retval += f"and will multiply by {self.reactive_multiplier} every 30s to a maximum of {self.gas_maximum}"
+        retval += f"and will multiply by {self.reactive_multiplier} every 30s to a maximum of " \
+                  f"{round(self.gas_maximum / self.GWEI, 1)} Gwei"
```

Comment:
Other gas amounts were shown in Gwei, so it made sense this should follow suit, especially because it should typically be quite a large number (in Wei).